Packs/Nmap/Integrations/Nmap/Nmap.yml

```
@@ -49,6 +49,12 @@ script:
       description: Any captured banner from the service
     - contextPath: NMAP.Scan.Hosts.Services.Service
       description: The service name
+    - contextPath: NMAP.Scan.Hosts.ScriptResults.ID
+      description: the name of the script used.
+    - contextPath: NMAP.Scan.Hosts.ScriptResults.Output
+      description: The raw results of the script execution
+    - contextPath: NMAP.Scan.Hosts.ScriptResults.Elements
+      description: May Include more parseable fields of the script output.
```

```suggestion
      description: The name of the script used.
    - contextPath: NMAP.Scan.Hosts.ScriptResults.Output
      description: The raw results of the script execution.
    - contextPath: NMAP.Scan.Hosts.ScriptResults.Elements
      description: Additional parseable fields from the script output.
```
not 100% I understood what are the elements outputs, please review my suggestion